oauth2client/browser.go

```diff
@@ -73,7 +73,7 @@ func (b Browser) Get(expectedStateVal, authCodeURL string) (string, error) {
 
 	err = openBrowser(authCodeURL)
 	if err != nil {
-		return "", err
+		fmt.Printf("Can't open browser (%s). Please follow the link: %s", err, authCodeURL)
```

Comment:
I know this is a petty nit, but what do you think about putting a line break, like this:

```suggestion
		fmt.Printf("Can't open browser: %s.\nPlease follow this link: %s", err, authCodeURL)
```

Reply:
Sure, thanks